README.md

```diff
@@ -48,57 +48,81 @@ We love and welcome contributions to our front-end repository which can be found
 npm install
 ```
 
-## 2. Set up Husky Hooks 🐕️
+<!-- ## 2. Set up Husky Hooks 🐕️
+
+/*This isn't implemented as a script anywhere */
 
 This will automatically run linting before commits and reduce common contributor mistakes.
 
 ```sh
 npm run prepare
-```
+``` -->
 
-## 3. Setting up the Database 🐘
+## 2. Setting up the Database 🐘
 
 For simplicity, we use firebase!
 
 For security, we run a local database so we don't break production!
 
 (optional) You can point to different database environments simply by changing the firebase url and variables in the `.env.dev` file.
 
-### 3.a Setup Firebase CLI
+### 2.a Setup Firebase CLI
 
 #### a.1 Install the Firebase CLI if you do not have it installed on your machine:
 
 ```sh
 npm -g i firebase-tools
 ```
 
-#### a.2 Install Java
+#### a.2 Install Java to the path
```

Comment:
Keep this install Java, since you need to have Java on your local machine to run the emulator